### ISSUE TITLE
Ways to help without coding

### ISSUE THREAD

**Issue by rathboma** [beekeeper-studio/beekeeper-studio#287]
Thank you for supporting Beekeeper Studio!

Plenty of you wonderful people have asked how to help but don't know how to code in Vue and Electron.

Fear not! There are lots of things you can do to help the project

## Help Spread the Word

1. Writing honest (we mean it!) reviews on [Slant.co](https://www.slant.co/options/38208/~beekeeper-studio-review), [G2](https://www.g2.com/products/beekeeper-studio/reviews), [Alternative To](https://alternativeto.net/software/beekeeper-studio/), or [Product Hunt](https://www.producthunt.com/posts/beekeeper-studio)
2. Tell your friends! Literally tell someone in person, or (if Covid is still around) via some digital mechanism like chat or Zoom.

## File bug reports

If you see something, say something (about bugs)! Please file an issue on this GitHub repository. No quirk is too small for a report. We care about the details.

## Make our documentation better

Did you learn how to do something cool in Beekeeper but it wasn't obvious on first try? You can contribute to our documentation! 

While our 'finished' docs are hosted on [docs.beekeeperstudio.io](https://docs.beekeeperstudio.io), the code is [right here in this repository](https://github.com/beekeeper-studio/beekeeper-studio/tree/master/docs) -- it's just regular old markdown files.

**Comment by xdiepx** [comment]
The ability to view store procedures and what the code looks like

**Comment by ridlafalah** [comment]
i love it, yet it isnt has a lot of features, if beekeeper studio can equal to dbeaver or even more, it can be the rissing stars, im already love the UI

**Comment by rathboma** [comment]
Thanks! That's the plan.

**Comment by ololosha228** [comment]
@rathboma can i copy part of your ideas into our repo with cc? It looks so helpful for literally **any** opensource project!

**Comment by rathboma** [comment]
Sure! No attribution required but a thank you link back is always appreciated :-)

**Comment by arnoldfederis** [comment]
Hello guys! I'm using this in my tutorial on how to connect in mysql using ubuntu wsl. I liked the UI and also I code using Vue and Laravel in my project. Cheers for this 👍 🍺 

Check it out guys! Thank you.
https://youtu.be/0emEc-QWhHI

**Comment by spacepetunia** [comment]
Do you have any plans to translate your documentation on other languages? Maybe, I can help your team to translate some guides for Beekeeper on Russian.

**Comment by rathboma** [comment]
That would be amazing! I am not able to translate myself, but the docs use VuePress, so it's pretty easy to create translations:

https://vuepress.vuejs.org/guide/i18n.html#site-level-i18n-config

Basically you create a new subdirectory with the translated docs in them.

**Comment by spacepetunia** [comment]
@rathboma in which branch I can create subdir and push translated docs?

**Comment by rathboma** [comment]
Not sure how used you are too GitHub, but you can 'fork' the repo to your own account and use any branch you like. Then open a pull request when you're ready

**Comment by UtechtDustin** [comment]
@rathboma I guess he meant "Which destination branch should i choose while creating the PR".
As the most Repositories dont instantly merge into the master, which is the case on this repository.
So the answer would be "master"  (i hope so).

**Comment by rathboma** [comment]
Yes! Master on this repo!

**Comment by Rajeeban-A** [comment]
> While our 'finished' docs are hosted on [docs.beekeeperstudio.io](https://docs.beekeeperstudio.io), the code is [right here in this repository](https://github.com/beekeeper-studio/beekeeper-studio/tree/master/docs) -- it's just regular old markdown files.
> 


please update the documentation repo URL

**Comment by ImagineItBetter** [comment]
> > , the code is [right here in this repository](https://github.com/beekeeper-studio/beekeeper-studio/tree/master/docs) -- it's just regular old markdown files.
> 
> please update the documentation repo URL

As stated by @Rajeeban-A above, /docs no longer exists on master, so the link results in a 404.

**Comment by dvasanin** [comment]
Please make it so that, whenever we apply a change to the db or commit a query, the refresh is automatic. Not having to disconnect from the database (example postgresql) if you're dropping tables or a database would be nice. Having to refresh the DB or disconnect every time I make a change is a pain and, as a fledgeling programmer, I make A LOT of changes. Imagine having to disconnect or refresh manually every time. 

And for the love of 1337, please make a changelog or a blog where we can track updates, see what new features you have put out.

**Comment by rathboma** [comment]
Hey @dvasanin! If you drop or change a table you can click the little 'refresh' button next to the entities in the sidebar, but soon we'll be automatically detecting those changes.b

If you drop a database you can click the 'refresh' button next to the database dropdown to do the same thing.

<img width="819" alt="Screen Shot 2021-09-23 at 10 27 21 AM" src="https://user-images.githubusercontent.com/279769/134537051-4b13838b-3bd6-4591-97a3-b15796b205fb.png">

Hope that helps!

In terms of change log, check out - https://www.beekeeperstudio.io/releases

**Comment by rathboma** [comment]
Also - I updated the docs link to make it correct. Sorry about that!

**Comment by Likenttt** [comment]
Are you planning to support i18n? If so, I can help with the zh-CN part

**Comment by rathboma** [comment]
@Likenttt at some point yes, although until I can support myself full time on the project I don't think it will be the priority.

**Comment by neil-xref** [comment]
> i love it, yet it isnt has a lot of features, if beekeeper studio can equal to dbeaver or even more, it can be the rissing stars, im already love the UI

I have dbeaver. What an atrocious UI. I don't want Beekeeper to try to emulate any particular feature set. I feel it's the clean and simple approach with the authors know and love that has generated such an awesome tool already. I used to keep dbeaver installed in case I needed it as well, but I never did. Now it's gone. Beekeeper is my only install for SQL apart from the command line tools like psql. Beekeeper is not a rising star, it is my SQL sun.

**Comment by PayamBijeh** [comment]
>  Beekeeper is not a rising star, it is my SQL sun.
I agree with you, when I was trying to learn about DB's, it was such a pain in the neck to find an editor for SQL or config my Vs code with any extension, and that was really shocking to me, I was like what the heck, do you mean we don't have any worthwhile SQL editor? until some guy introduced beekeeper to me and I owe him a big thanks :)

**Comment by LO-teknologi** [comment]
My IP seems to be blocked from accessing testimonial.to, but I just want to leave something because Beekeeper makes me feel excited to work in data again. Haven't felt like this in years, working day in and day out with GCP's deranged interface and DBeaver's overwrought features. Cut the hours spent in tedious data checking in half, the layout makes me want to actually explore the data. Finally something that looks like it's made by someone actually working with databases. I'll be telling my fellow analysts about this. 

Thank you very much. It's amazing how much difference a well-designed tool can make.

Now, if I can have a shortcut for "check and count all distinct values in this column" and "show min and max value for this column" I'll be a happy clam.
